site/it/tutorials/keras/classification.ipynb

```diff
@@ -563,7 +563,7 @@
       "source": [
         "Il primo livello in questa rete, `tf.keras.layers.Flatten`, trasforma il formato delle immagini da un array bi-dimensionale (di 28 per 28 pixel) in un array uni-dimensionale (di 28 * 28 = 784 pixel). Pensate a questi livelli come righe non impilate di pixel dell'immagine. Questo livello non ha parametri da imparare; esso si limita a rifirmattare i dati.\n",
         "\n",
-        "Dopo la normalizzazione dei pixel, la rete consiste di due livelli `tf.keras.layers.Dense`. Questi sono livelli neurali strettamente connessi, o completamente connessi. Il primo livello `Denso` ha 128 nodi  (o neuroni). Il secondo (ed ultimo) livello è un livello *softmax* a 10 nodi che restituisce un vettore di 10 valori di probabilità la cui somma è 1. Ogni nodo contiene un valore che indica la probabilità che l'immagine corrente appartenga ad una delle 10 classi.\n",
+        "Dopo la normalizzazione dei pixel, la rete consiste di due livelli `tf.keras.layers.Dense`. Questi sono livelli neurali strettamente connessi, o completamente connessi. Il primo livello `Denso` ha 128 nodi (o neuroni). Il secondo (ed ultimo) livello è un livello *softmax* a 10 nodi che restituisce un vettore di 10 valori di probabilità la cui somma è 1. Ogni nodo contiene un valore che indica la probabilità che l'immagine corrente appartenga ad una delle 10 classi.\n",
```

Comment:
Densamente o strettamente?

Reply:
Personally I'm not sure if we specifically use one or the other in this situation ^^
The English sentence there is "These are densely connected, or fully connected, neural layers."
For this maybe "densamente" is better but maybe "strettamente" is more appropriated here.
@codeadmin-peritiae can we ask you if we should change strettamente to densamente or if it's fine like this?

Reply:
Search "densamente" in http://www.treccani.it/enciclopedia/storia-dei-concetti-e-delle-tecniche-nella-ricerca-sulle-reti-neurali_%28Frontiere-della-Vita%29/